docs/architecture/adr-069-gov-improvements.md

```diff
@@ -0,0 +1,201 @@
+
+Those two types are, namely: multiple choice proposals and optimistic proposals.
+
+## Context
+
+`x/gov` is the center of Cosmos governance, and has already been improved from its first version `v1beta1`, with a second version [`v1`][5].
+This second iteration on gov unlocked many possibilities by letting governance proposals contain any number of proposals.
+The last addition of gov has been expedited proposals (proposals that have a shorter voting period and a higher quorum, approval threshold).
+
+The community requested ([1], [4]) two additional proposals for improving governance choices. Those proposals would be useful when having protocol decisions made on specific choices or simplifying regular proposals that do not require high community involvement.
+
+Additionally, the SDK should allow chains to customize the tallying method of proposals (if they want to count the votes in another way). Currently, the Cosmos SDK counts votes proportionally to the voting power/stake. However, custom tallying could allow counting votes with a quadratic function instead.
+
+## Alternatives
+
+N/A
+
+## Decision
+
+`x/gov` will integrate those functionalities and extract helpers and interfaces for extending the `x/gov` module capabilities.
+
+### Proposals
+
+Currently, all proposals are [`v1.Proposal`][5]. Optimistic and multiple choice proposals only require a different tally logic, but the rest of the proposal stays the same to not create other proposal types, `v1.Proposal` will have an extra field:
+
+```protobuf
+enum ProposalType {
+  // NORMAL defines a normal propo
+  NORMAL = 0;
+  // VOTE_OPTION_YES defines a yes vote option.
+  MULTIPLE_CHOICE = 1;
+  // VOTE_OPTION_ABSTAIN defines an abstain vote option.
+  OPTIMISTIC = 2;
+  // VOTE_OPTION_NO defines a no vote option.
+  EXPEDITED = 3;
+}
+```
+
+Note, that expedited becomes a proposal type itself instead of a boolean on the `v1.Proposal` struct.
+
+> An expedited proposal is by design a normal proposal with a quicker voting period and higher threshold. A expedited proposals if fail, gets converted to a normal proposal
+
+An expedited optimistic proposal and an expedited multiple choice proposal do not make sense based on the definition above and is a proposal type instead of a proposal characteristic.
+
+#### Optimistic Proposal
+
+An optimistic proposal is a proposal that automatically passes unless a threshold a NO votes is reached.
+
+Voter can only vote NO on the proposal. If the NO threshold is reached, the optimistic proposal is converted to a normal proposal.
+
+Two governance parameters will be in added [`v1.Params`][5] to support optimistic proposals:
+
+```protobuf
+// optimistic_authorized_addreses is an optional governance parameter that limits the authorized accounts than can submit optimisitc proposals
+repeated string optimistic_authorized_addreses = 17 [(cosmos_proto.scalar) = "cosmos.AddressString"];
```

Comment:
What's the expected usage around this being initially set? Who do you imagine being in this list?

Reply:
A list of authorized address to not allow everyone to submit an optimistic proposal, if wished by the chain.